handsontable/handsontable
# ContextMenu's sub-instance shouldn't leave its listeners.

### Context
After these changes, ContextMenu's items list will collect items on demand - before open ContextMenu.

### How has this been tested?
* Use configuration from a demo: https://jsfiddle.net/1m8q2fu6/
* Click several times on the "updateSettings" button.
* Click on the "destroy" button.
* Check in your favourite dev tools, how many `mousedown` listeners are still listening on `document.documentElement` (`html` node).

### Types of changes

### Related issue(s):
1. #5759
